code/modules/clothing/masks/necklaces.dm

```diff
@@ -2,6 +2,130 @@
 	body_parts_covered = 0
 	ignore_flip = TRUE
 
+/obj/item/clothing/mask/explosive_collar
```

Comment:
Do you want mechanics to be able to scan their own death devices?

Reply:
It's not scannable

Reply:
also it's not a death device it's a cool necklace